Log4swift/Logger.swift

```diff
@@ -50,6 +50,7 @@ A logger is identified by a UTI identifier, it defines a threshold level and a d
   private var thresholdLevelStorage: LogLevel
   private var appendersStorage: [Appender]
   private var asynchronousStorage = false
+  private var timeProviderStorage: (() -> Date)?
```

Comment:
To avoid having an optional value, may I suggest to change the declaration to this:
`private var timeProviderStorage: (() -> Date) = Date.init`

It would notably remove the doubled default value on lines 263 and 292.

Reply:
Good idea - see updated PR.